Week-08/GEOS639-Lab4-SBASInSARUsingMitPy.ipynb

```diff
@@ -1537,9 +1537,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "unavco",
```

Comment:
These is the right `display_name` and `name`, correct?

Reply:
Yes, we changed it to just `unavco` few weeks back when I was working with Alex.